plugin.php

```diff
@@ -205,15 +212,27 @@ function save_contents_for_url( $contents, $url, $config = null ) {
 	do_action( 'static_page_saved_contents_for_url', $path, $config );
 }
 
-function remove_url( $url, $config = null ) {
+/**
+ * @param $url
+ * @param null $config
+ * @param array $option_args
+ */
+function remove_url( $url, $config = null, $option_args = array() ) {
```

Comment:
@dan-westall Should we check to see if `$url` has been set and if it hasn't just return? I think I've seen an error in the logs when the URL hasn't been set or changed.